src/test/java/org/apache/commons/io/build/CharSequenceOriginTest.java

```diff
@@ -84,4 +84,13 @@ public void testGetWriter() {
         assertThrows(UnsupportedOperationException.class, super::testGetWriter);
     }
 
+    @Test
+    public void testGetReader_UTF16() throws IOException {
```

Comment:
Doesn't this test the same behavior as 'testGetReaderIgnoreCharset()?

Reply:
Yes. I saw your comment asking for a test before I saw you had added one.